### ISSUE TITLE
Fix(UI): No Padding between the components

### ISSUE THREAD

**Issue by narainkarthikv** [narainkarthikv/fit-track#38]
Resolve the issue of removing padding between UI components to enhance layout spacing and visual appeal.

## **Tasks**  

- [x] Remove padding between components to ensure a consistent and visually pleasing layout.  
- [x] Verify that the changes align with the overall design and spacing guidelines.

**Comment by HTMLSlander** [comment]
Hello! I know its late but the issue still persists? or nah

**Comment by HTMLSlander** [comment]
Yeah sure you can assign me with this one

**Comment by HTMLSlander** [comment]
bruh log in sign up doesnt work and i cant get access to home 💀

**Comment by narainkarthikv** [comment]
> bruh log in sign up doesnt work and i cant get access to home 💀

Go through this readme for getting the setup of application of both frontend and backend [`Readme.md`](https://github.com/narainkarthikv/fit-track/blob/develop/README.md)

**Comment by HTMLSlander** [comment]
yeah i already did that but how doyou go to home page without loading in

**Comment by narainkarthikv** [comment]
> yeah i already did that but how do you go to home page without loading in

Have you tried setting up the backend with your own cluster and DB setup and create a account using your own email and password?

**Comment by HTMLSlander** [comment]
Yeah I already set up the backend and front end part. Then, tried to create account but It seems that this doesnt work, bc the DOM displays a "Passwords do not match" message, although i put the same thing for each password input. Also tried with some fake users section and didn't worked anyways.

<img width="939" alt="Image" src="https://github.com/user-attachments/assets/591f32d3-c930-456b-a7f7-8ca82b671d1a" />

**Comment by narainkarthikv** [comment]
Hey @HTMLSlander , I've commited new changes to develop branch. Can you now check it, now? Now the issue is resolved and I tested it in development server. Let me know if you face any issues.

[Fit Track Dev server](https://fit-track-dev.netlify.app/)

**Comment by HTMLSlander** [comment]
yeah, it's working for me, thanks for commiting!

**Comment by HTMLSlander** [comment]
It works on netlify but when I set up on localhost the Sign up fails to sign although i put my mongodb gmail and password.

**Comment by narainkarthikv** [comment]
> It works on netlify but at the local host when i Sign up fails to sign although i put my mongodb gmail and password.

I think now the signup works as I've recently commited changes in `develop` branch. Can you pull the changes and work on it?

**Comment by narainkarthikv** [comment]
Issue completed by @HTMLSlander
